webinars/net-on-rhel-sneak-peek.html.slim

```diff
@@ -0,0 +1,25 @@
+---
+layout: base
+body_class: fullbleed
+primary_nav_hidden: true
+interpolate: true
+drupal_type: rhd_microsite
+---
+
+.microsite-header#net-rhel-sneakpeak
+  .row
+    .large-8.medium-11.small-24.columns
+      img.dotnet-graphic(src=cdn("#{site.base_url}/images/events/msbuild/dotNETwebinar_promotiongraphic.png") alt=".NET on RHEL Sneak Peak Webinar")
+      
+    .large-16.medium-13.small-24.columns
+      h3.caps Behind-the-scenes of .NET on Red Hat Enterprise Linux
+      p
+        | You may have heard Microsoft hearts Linux.  And soon you will be able to use .NET on Red Hat Enterprise Linux. While the pieces are still being finalized we are excited to share a behind-the-scenes sneak peek at how this will work for you.
+      p
+        | In this 2-hour webinar and demo, see:
+        ul
+          li How .NET on Red Hat Enterprise Linux is coming together
+          li What you can expect from the development experience
+          li .NET on Red Hat Enterprise Linux in action
+        strong Register now for this unique sneak peek.
+      a.button.long(href="#") Register Now
```

Comment:
This button doesn't do anything. It should go to the 'registration-confirmation' page.